mm2src/mm2_test_helpers/src/structs.rs

```diff
@@ -904,20 +904,28 @@ pub struct CoinAddressInfo<Balance> {
 
 pub type TokenBalances = HashMap<String, CoinBalance>;
 
+#[derive(Debug, Deserialize)]
+#[serde(deny_unknown_fields)]
+pub struct EnableEthWithTokensResponse {
+    pub current_block: u64,
+    pub eth_addresses_infos: Option<HashMap<String, CoinAddressInfo<CoinBalance>>>,
```

Comment:
I have to use the same name as the one in the original struct https://github.com/KomodoPlatform/atomicDEX-API/blob/19fd34e3bfff91ab1fb9bd395a5df68b71f306ae/mm2src/coins_activation/src/eth_with_token_activation.rs#L151
And this name shouldn't be changed since it's already being used in GUIs.

Reply:
optional: Yeah, this sad, you're right of coarse. It's up to you I would use [serde(rename ..)]. Mark it as read please, and I resolve this thread then